src/helics/apps/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@
 
 if(HELICS_BUILD_APP_LIBRARY)
 
+    cmake_dependent_option(
+        HELICS_ENABLE_WEBSERVER "Enable a webserver app structure to be embedded with the broker server" OFF
+        "NOT HELICS_DISABLE_BOOST;Boost_VERSION VERSION_GREATER 1.69" OFF
```

Comment:
It looks like Boost didn't release any 1.69.x versions, so `Boost_VERSION VERSION_GREATER 1.69` is probably okay.

Reply:
I am kind of looking forward to being able to use `VERSION_GREATER_EQUAL` with HELICS 3.0